lib/irb.rb

```diff
@@ -850,12 +805,6 @@
 #
 #   IRB.conf[:AP_NAME] = 'my_ap_name'
 #
-# :stopdoc:
-# === \IRB Library Directory
```

Comment:
This config is only used by the current tracer integration to skip IRB related traces. We don't need to disclose it in the doc.